namerd/storage/consul/src/test/scala/io/buoyant/namerd/storage/consul/ConsulDtabStoreTest.scala

```diff
@@ -0,0 +1,70 @@
+package io.buoyant.namerd.storage.consul
+
+import com.twitter.finagle.http.{Request, Response, Status}
+import com.twitter.finagle.{Path, Service}
+import com.twitter.io.Buf
+import com.twitter.util.{Activity, Future}
+import io.buoyant.consul.v1.KvApi
+import io.buoyant.namerd.Ns
+import io.buoyant.test.{Awaits, Exceptions, FunSuite}
+
+class ConsulDtabStoreTest extends FunSuite with Awaits with Exceptions {
```

Comment:
it doesn't look like this tests the invalid namespace check at all.  that check applies to the create, delete, put, and get methods.

Reply:
I can tests for that functionality. I was under the impression that we were trying to get this out the door by today so I just wrote basic tests for the ConsulDTabStore